<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-94IKA2I" failures="1" tests="7" name="testjava.LoginPageTest" time="25.590" errors="0" timestamp="2022-06-30T15:09:15 IST" skipped="0">
  <testcase classname="testjava.LoginPageTest" name="verifyTitle" time="0.052">
    <failure message="expected [OrangeHRMa] but found [OrangeHRM]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [OrangeHRMa] but found [OrangeHRM]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testjava.LoginPageTest.verifyTitle(LoginPageTest.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
    </failure>
  </testcase> <!-- verifyTitle -->
  <system-out/>
  <testcase classname="testjava.LoginPageTest" name="verifyURL" time="0.013"/>
  <system-out/>
  <testcase classname="testjava.LoginPageTest" name="verifyLogin" time="4.692"/>
  <system-out>
Verify Login with Correct cre  </system-out>
  <testcase classname="testjava.LoginPageTest" name="verifyLogin" time="6.776"/>
  <system-out>
Verify Login with Correct cre  </system-out>
  <testcase classname="testjava.LoginPageTest" name="verifyLogin" time="4.728"/>
  <system-out>
Verify Login with Correct cre  </system-out>
  <testcase classname="testjava.LoginPageTest" name="verifyLogin" time="4.914"/>
  <system-out>
Verify Login with Correct cre  </system-out>
  <testcase classname="testjava.LoginPageTest" name="verifyLogin" time="4.415"/>
  <system-out>
Verify Login with Correct cre  </system-out>
</testsuite> <!-- testjava.LoginPageTest -->
